src/frontend/src/lib/components/core/Background.svelte

```diff
@@ -7,7 +7,7 @@
 
 <div class="fixed inset-0 z-[-1] overflow-hidden bg-white">
 	<picture
-		class="absolute left-1/2 top-0 w-full min-w-[1280px] -translate-x-1/2 transform"
+		class="absolute left-1/2 top-0 w-full min-w-[1728px] -translate-x-1/2 transform"
```

Comment:
We cannot use something like min width 100vw or something?

Reply:
the idea is that they don't want it to re-size below the inherent width of the SVG (1728px): in those cases it should just hide in overflow. However, it can enlarge for bigger screens.

Reply:
Should we create a new breakpoint in the tailwind configuration for it that way we don't have an hardcoded value?

Reply:
what if instead of a breakpoint, I make a custom `minWidth`? 

```
// tailwind config
extend: {
	minWidth: {
		'2_5xl': '1728px'
	}
}

// code
... w-full min-w-2_5xl ...
```

Reply:
sounds good too!

Reply:
done!